packages/bitcore-lib/lib/transaction/sighash.js

```diff
@@ -127,29 +115,15 @@ function sign(transaction, privateKey, sighashType, inputIndex, subscript, signi
  * @param {PublicKey} publicKey
  * @param {number} inputIndex
  * @param {Script} subscript
- * @param {String} signingMethod - method used to sign - 'ecdsa' or 'schnorr'
  * @return {boolean}
  */
-function verify(transaction, signature, publicKey, inputIndex, subscript, signingMethod) {
+function verify(transaction, signature, publicKey, inputIndex, subscript) {
```

Comment:
Consider updating method call params to reflect new arguments:

- Input.prototype.isValidSignature
- MultiSigInput.prototype.isValidSignature
- MultiSigInput.normalizeSignatures
- MultiSigScriptHashInput.prototype.isValidSignature
- PublicKeyHashInput.prototype.isValidSignature